src/frontend/components/PlaylistPortability/index.tsx

```diff
@@ -0,0 +1,275 @@
+                    removePlaylistPortability(item.id);
+                  }}
+                />
+              )}
+            >
+              {(reachableFromPlaylist: { title: string; id: string }) => (
+                <Box
+                  role="listitem"
+                  a11yTitle={intl.formatMessage(
+                    messages.hasPortabilityWith,
+                    reachableFromPlaylist,
+                  )}
+                  direction="row-responsive"
+                  gap="medium"
+                  align="center"
+                >
+                  <Text weight="bold">{reachableFromPlaylist.title}</Text>
+                  <Text size="small" color="dark-4">
+                    {reachableFromPlaylist.id}
+                  </Text>
+                </Box>
+              )}
+            </List>
+          </Box>
+        );
+      }
+
+      content = (
+        <React.Fragment>
+          <Box align="center" direction="row" pad={{ top: 'small' }}>
+            <Text role="heading" margin="small">
+              <FormattedMessage
+                {...messages.playlistTitle}
+                values={{ title: playlist?.title, id: playlist?.id }}
+              />
+              <Button
+                aria-label={`copy key ${playlist?.id}`}
+                data-clipboard-text={playlist?.id}
+                icon={<Copy />}
+                className="copy"
+              />
+            </Text>
+          </Box>
+
+          <Box width="large">
+            {portabilityList}
+            <Box>
+              <Form onSubmit={addPlaylistPortability}>
+                <FormField
+                  label={<FormattedMessage {...messages.shareWithPlaylist} />}
+                  htmlFor="port-to-playlist-uuid"
+                >
+                  <Box direction="row">
+                    <TextInput
+                      placeholder={intl.formatMessage(
+                        messages.shareWithPlaylistPlaceholder,
+                      )}
+                      id="port-to-playlist-uuid"
+                      onChange={(event) =>
+                        setNewPortabilityID(event.target.value)
+                      }
+                      value={newPortabilityID}
+                      plain
+                    />
+                    <Button
+                      type="submit"
+                      aria-label="add share"
+                      icon={<AddCircle />}
+                    />
+                  </Box>
+                </FormField>
+              </Form>
+            </Box>
+
+            <Box
+              align="center"
+              margin={{ top: 'medium' }}
+              pad="small"
+              background="status-warning"
+              width="large"
+            >
+              <Text>
+                <FormattedMessage {...messages.shareWithPlaylistDetails} />
+              </Text>
+            </Box>
+          </Box>
+        </React.Fragment>
+      );
+      break;
+  }
+
+  return (
```

Comment:
All this return looks like a lot what is made in the `DashboardContainer`. You should maybe create a component dedicated to display the dashboardContainer, the `LTINav` and a `children`. A generic component

```
<DashboardContainer>
      <LTINav object={object} />
      {children}
</DashboardContainer>
```